### PR TITLE
Fixed an issue with preview courses URL and ORA urls in devstack

### PR DESCRIPTION
**Pull request by OmarIthawi**
This is something I need while developing.

This fixes: 

 - The ORA2 problems do not work locally.
 - The "view live" button on studio goes to the incorrect URL.

---

This is a reopen for #197

### CODE REVIEW

**Comment by OmarIthawi** [comment]
@ahmedaljazzar could you please review this? It's a devstack PR that you'll like to have on the platform.

File: cms/envs/devstack.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 ################################# LMS INTEGRATION #############################
 
-LMS_BASE = "localhost:8000"
+# LMS_BASE = "localhost:8000"  # Edraak: Commented out because we use domain names locally.
```
Comment (ahmedaljazzar) [inline cms/envs/devstack.py]: And what the base should look like? Isn't it better to get it from the envs file rather than commenting it out?

Comment (OmarIthawi) [inline cms/envs/devstack.py]: That's the thing, this one overrides the JSON value that `cms/envs/aws.py` provides. Commenting it out makes things right. Perhaps the comment is misleading. Could you please double check on your machine to see whether the `LMS_BASE` value is already set to the correct value: `courses.edraak.dw`.

Comment (ahmedaljazzar) [inline cms/envs/devstack.py]: No it's not.

Comment (OmarIthawi) [inline cms/envs/devstack.py]: I'll have to check with you.

**Pull request merged**